target/linux/realtek/files-5.10/drivers/net/dsa/rtl83xx/rtl838x.c

```diff
@@ -990,17 +990,17 @@ static void rtl838x_read_pie_templated(u32 r[], struct pie_rule *pr, enum templa
 			pr->sip_m = (pr->sip << 16) | data_m;
 			break;
 		case TEMPLATE_FIELD_SIP2:
+		case TEMPLATE_FIELD_SIP3:
+		case TEMPLATE_FIELD_SIP4:
+		case TEMPLATE_FIELD_SIP5:
+		case TEMPLATE_FIELD_SIP6:
+		case TEMPLATE_FIELD_SIP7:
 			pr->is_ipv6 = true;
 			// Make use of limitiations on the position of the match values
 			ipv6_addr_set(&pr->sip6, pr->sip, r[5 - i / 2],
 				      r[4 - i / 2], r[3 - i / 2]);
 			ipv6_addr_set(&pr->sip6_m, pr->sip_m, r[5 - i / 2],
 				      r[4 - i / 2], r[3 - i / 2]);
-		case TEMPLATE_FIELD_SIP3:
-		case TEMPLATE_FIELD_SIP4:
-		case TEMPLATE_FIELD_SIP5:
-		case TEMPLATE_FIELD_SIP6:
-		case TEMPLATE_FIELD_SIP7:
 			break;
```

Comment:
This is not really a cosmetic change, and you have a couple more of these. Just an oversight, or is there a reason you included these changes?

Maybe {SIP,DIP}3-7 aren't actually used, so _currently_ this wouldn't make any difference.

Reply:
they should be moved to the top, e.g. first do all the case labels, then do the case content iirc. I did this in the cosmetic patch because I have a later patch that does add the 'fallthrough' label. it COULD be that i messed up a rebase though; so I'll double check :) But i specifically tried to put all the non-code/style changes in this commit.